lib/handler.js

```diff
@@ -66,7 +66,7 @@ internals.prerequisites = function (request, callback) {
 internals.handler = function (request, callback) {
 
     var timer = new Hoek.Bench();
-    var finalize = function (response, data) {
```

Comment:
I rather keep `data` here because it's part of the reply interface callback spec even if it is not used.